content/docs/addons.md

```diff
@@ -4,46 +4,45 @@ title: Add-Ons
+Os add-ons abaixo são considerados legados e seu uso é desencorajado. Eles continuarão trabalhando em um futuro observável, mas não há mais desenvolvimento.
 
-- [`PureRenderMixin`](/docs/pure-render-mixin.html). Use [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
-- [`shallowCompare`](/docs/shallow-compare.html), a helper function that performs a shallow comparison for props and state in a component to decide if a component should update. We recommend using [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
-- [`update`](/docs/update.html). Use [`kolodny/immutability-helper`](https://github.com/kolodny/immutability-helper) instead.
-- [`ReactDOMFactories`](https://www.npmjs.com/package/react-dom-factories), pre-configured DOM factories to make React easier to use without JSX.
+- [`PureRenderMixin`](/docs/pure-render-mixin.html). Utilize [`React.PureComponent`](/docs/react-api.html#reactpurecomponent).
+- [`shallowCompare`](/docs/shallow-compare.html), uma função auxiliar que executa uma comparação superficial para props e state em um componente para decidir se um componente deve atualizar. Nós recomendamos utilizar [`React.PureComponent`](/docs/react-api.html#reactpurecomponent).
+- [`update`](/docs/update.html). Utilize [`kolodny/immutability-helper`](https://github.com/kolodny/immutability-helper).
+- [`ReactDOMFactories`](https://www.npmjs.com/package/react-dom-factories), fábricas DOM pré-configuradas para fazer o React ser mais fácil de utilizar sem JSX.
 
-### Deprecated Add-ons {#deprecated-add-ons}
+### Add-ons Descontinuados {#deprecated-add-ons}
 
-- [`LinkedStateMixin`](/docs/two-way-binding-helpers.html) has been deprecated.
-- [`TransitionGroup` and `CSSTransitionGroup`](/docs/animation.html) have been deprecated in favor of [their drop-in replacements](https://github.com/reactjs/react-transition-group/tree/v1-stable).
+- [`LinkedStateMixin`](/docs/two-way-binding-helpers.html) foi descontinuado.
+- [`TransitionGroup` and `CSSTransitionGroup`](/docs/animation.html) foi descontinuado em favor de [suas substituições drop-in](https://github.com/reactjs/react-transition-group/tree/v1-stable).
 
-## Using React with Add-ons {#using-react-with-add-ons}
+## Utilizandos React com Add-ons {#using-react-with-add-ons}
 
-You can install the add-ons individually from npm (e.g. `npm install react-addons-create-fragment`) and import them:
+Você pode instalar os add-ons individualmente via npm (e.g. `npm install react-addons-create-fragment`) e importá-lo:
 
 ```javascript
 import createFragment from 'react-addons-create-fragment'; // ES6
 var createFragment = require('react-addons-create-fragment'); // ES5 with npm
 ```
-
-When using React 15 or earlier from a CDN, you can use `react-with-addons.js` instead of `react.js`:
+Quando utilizar React 15 ou anterior via CDN, você pode usar `react-with-addons.js` ao invés de `react.js`:
```

Comment:
Revolveu uma linha, isso pode atrapalhar o bot quando for dar merge com as novas alterações da documentação principal.
```suggestion

Quando utilizar React 15 ou anterior via CDN, você pode usar `react-with-addons.js` ao invés de `react.js`:
```